nginx_stage/templates/pun.conf.erb

```diff
@@ -50,6 +50,10 @@ http {
   # Load all apps directly
   passenger_spawn_method direct;
 
+  <%- passenger_options.to_h.each_pair do |key, value| -%>
+    passenger_<%= key %> <%= value %>;
+  <%- end -%>
```

Comment:
Seems like we should follow the pattern in `pun_custom_env` where we have a writer and in the reader we just reject anything that doesn't match the pattern `passenger_.+`.  It may be confusing for folks if we're prepending passenger to the keys.

Of course it's a more complex task to reject anything we already currently support but we'll just have to cross that when we get there.

Reply:
I updated so we only accept the options that begin with `passenger_`.

If we want to reject options already set, I think we'd have to define a key/value pair of options we always set, and use that hash to populate the template rather than hardcoding the values into the template.  That way the same hash could be used to reject existing values we hardcode.